InnerEye/ML/configs/other/HelloContainer.py

```diff
@@ -30,15 +30,22 @@ class HelloDataset(Dataset):
     # y = 0.2 * x + 0.1 * torch.randn(x.size())
     # xy = torch.cat((x, y), dim=1)
     # np.savetxt("Tests/ML/test_data/hellocontainer.csv", xy.numpy(), delimiter=",")
-    def __init__(self, root_folder: Path, start_index: int, end_index: int) -> None:
+    def __init__(
+            self,
+            root_folder: Path,
+            start_index: int,
+            end_index: int,
+            raw_data: List = None) -> None:
         """
         Creates the 1-dim regression dataset.
         :param root_folder: The folder in which the data file lives ("hellocontainer.csv")
         :param start_index: The first row to read.
         :param end_index: The last row to read (exclusive)
+        :param raw_data: (Optional, default None) if supplying the raw data, e.g. from a cross validation split
```

Comment:
I don't get that docstring.

Reply:
I don't like the two ways this class is constructed, in another language it would have two constructors, but we cannot do that in Python. I'll refactor.

Reply:
Done